### ISSUE TITLE
Custom template in openapi 3 doesnt work always throws java.io.FileNotFoundException: /C:\user\template/C:\user\template\model.mustache

### ISSUE THREAD

**Issue by anand188** [swagger-api/swagger-codegen#8132]
Exception in thread "Thread-1" java.lang.RuntimeException: Could not generate model 'AllLinks'
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:407)
        at io.swagger.codegen.DefaultGenerator.generate(DefaultGenerator.java:722)
        at io.swagger.codegen.cmd.Generate.run(Generate.java:312)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.io.FileNotFoundException: /C:\user\template/C:\user\template\model.mustache
        at com.github.jknack.handlebars.io.URLTemplateLoader.sourceAt(URLTemplateLoader.java:70)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:357)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:343)
        at io.swagger.codegen.DefaultGenerator.getHandlebars(DefaultGenerator.java:1017)
        at io.swagger.codegen.DefaultGenerator.processTemplateToFile(DefaultGenerator.java:738)
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:394)

**Comment by anand188** [comment]
@HugoMario is there any latest jar published resolves this issue

**Comment by robert2411** [comment]
C:\user\template/C:\user\template\model.mustache
i see 2 times the beginning of the path in here are you sure your path is set correctly

**Comment by HugoMario** [comment]
Hello @anand188, sorry for this really late response, i'm wondering if you got this issue trying with latest changes on `3.0.0-SNAPSHOT` branch

**Comment by anand188** [comment]
@HugoMario  3.0.0-SNAPSHOT do you refer to pom.xml ?    
<groupId>io.swagger</groupId>
                <artifactId>swagger-codegen-maven-plugin</artifactId>
                <version>3.0.0-rc0</version>

Above only i am using which doesn't work

**Comment by anand188** [comment]
@HugoMario 3.0.0-SNAPSHOT also you specify it as branch i only see  3.0.0 branch .
If any branch regarding snapshot is there ?

**Comment by anand188** [comment]
@HugoMario i tested the same now it was not working i am working in windows .
whats your OS? and is code tested in all OS?

**Comment by HugoMario** [comment]
@anand188 can you please share the command you're using?

**Comment by anand188** [comment]
@HugoMario the below is the command i used
 java -jar swagger-codegen-cli-3.0.0-SNAPSHOT.jar generate -i authorization.yaml -l jaxrs-jersey -c config.json -t C:\anand\template

**Comment by HugoMario** [comment]
ok, FYI checking in a windows in a bit

**Comment by bigtlb** [comment]
I am also having issues with the template option in Windows.

If I use forward slashes in the path, I get File Not Found and duplicated part of the path (like above).  If I use backslashes, like:

 `-t C:\Users\mrtlb\template`

I get:

```
Caused by: java.util.regex.PatternSyntaxException: Illegal/unsupported escape sequence near index 3
C:\Users\mrtlb\template
  ^
```

**Comment by HugoMario** [comment]
This isse has been fixed by #8284 and #8246

**Comment by OnkelTem** [comment]
Now it doesn't work on Linux too. But the reason is different.

https://github.com/swagger-api/swagger-codegen/issues/10107
